### DISCUSSION TITLE
为啥更新之后在应用那里一个应用都没有了

### DISCUSSION THREAD

**Discussion by liqiaozhiaofei** [gkd-kit/gkd#761] [General]
![Screenshot_2024-11-01-09-40-00-910_li.songe.gkd.jpg](https://github.com/user-attachments/assets/3ddad5ed-db76-4492-a5e2-26896a2616dc)

**Comment by lisonge** [comment]
![img_v3_02g7_52d4236c-ed88-4a83-bba9-41fe42e0cb3g](https://github.com/user-attachments/assets/e403818b-a3a3-402b-9ffb-fd683be3d94f)

我不知道为什么你的设备不显示这个权限提示，你可以把日志发出来

**Comment by liqiaozhiaofei** [comment]
日志来了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月1日(周五) 上午10:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [gkd-kit/gkd] 为啥更新之后在应用那里一个应用都没有了 (Discussion #761)




 
img_v3_02g7_52d4236c-ed88-4a83-bba9-41fe42e0cb3g.jpg (view on web)
 
我不知道为什么你的设备不显示这个权限提示，你可以把日志发出来

**Comment by liqiaozhiaofei** [comment]
我卸载重装之后有那个权限显示了
&nbsp;


行
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月1日(星期五) 上午10:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [gkd-kit/gkd] 为啥更新之后在应用那里一个应用都没有了 (Discussion #761)














 
img_v3_02g7_52d4236c-ed88-4a83-bba9-41fe42e0cb3g.jpg (view on web)
 
我不知道为什么你的设备不显示这个权限提示，你可以把日志发出来

**Reply by dc0358** [reply]
重启手机，并把读取应用权限设置为始终允许即可
